package.json

```diff
@@ -54,6 +54,7 @@
     "flatpickr": "^4.6.13",
     "html-react-parser": "^5.1.12",
     "react-modal": "^3.16.1",
+    "react-quick-pinch-zoom": "^5.1.0",
```

Comment:
this isn't necessarily an indictment of this package, but I do see a much more widely used react zoom package according to npmtrends: 
![image](https://github.com/user-attachments/assets/dde622dc-b61f-4d44-b4ef-e04d546485dd)
Did you have a chance to check that one out?